.github/workflows/fpga.yml

```
@@ -290,6 +304,10 @@ jobs:
         uses: actions/checkout@v3
         with:
           submodules: 'true'
+
+      - name: Pull dpe submodule
+        run: |
+          git submodule update --init dpe
```

```suggestion
          submodules: 'recursive'
```

Looks like a few places could use this pattern to remove a step.